P5/Source/Specs/glyph.xml

```diff
@@ -1,75 +1,60 @@
 <?xml version="1.0" encoding="UTF-8"?><!--
-Copyright TEI Consortium. 
-Dual-licensed under CC-by and BSD2 licences 
+Copyright TEI Consortium.
+Dual-licensed under CC-by and BSD2 licences
 See the file COPYING.txt for details
 $Date$
 $Id$
---><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?><elementSpec xmlns="http://www.tei-c.org/ns/1.0" module="gaiji" xml:id="GLYPH" ident="glyph">
+--><?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec ident="glyph" module="gaiji" xml:id="GLYPH" xmlns="http://www.tei-c.org/ns/1.0" validUntil="2021-07-09">
+  <desc versionDate="2019-07-01" type="deprecationInfo" xml:lang="en">The use of this element is deprecated, and will be considered invalid after 2021-07-09. Use either `unicodeProp`, `localProp`, or `unihanProp` to specify character or glyph properties.</desc>
```

Comment:
```suggestion
  <desc versionDate="2019-07-01" type="deprecationInfo" xml:lang="en">Use either <gi>unicodeProp</gi>, <gi>localProp</gi>, or <gi>unihanProp</gi> to specify character or glyph properties.</desc>
```

Reply:
Hold it. Is the `<glyph>` element going away? I did not think so. If not, remove the `@validUntil` on the `<elementSpec>`. We also may need to add a `<constraintSpec type="deprecationWarning">` about the content.